tests/tiramisu/test_schedule.py

```diff
@@ -13,7 +13,7 @@ def test_execute():
     schedule = Schedule(test_program)
     assert schedule.tree
     schedule.add_optimizations([Parallelization(params=[("comp02", 0)])])
-    results = schedule.execute(nb_exec_times=10)
+    results = schedule.execute(min_runs=10)
```

Comment:
Would you please add some test cases for the different types of usage?
It will serve as both tests and usage examples.
Thank you